src/metaschema/oscal_mapping_metaschema.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../../build/metaschema/toolchains/xslt-M4/validate/metaschema-composition-check.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<!-- OSCAL CATALOG METASCHEMA -->
+<!-- validate with XSD and Schematron (linked) -->
+<!DOCTYPE METASCHEMA [
+   <!ENTITY allowed-values-control-group-property-name SYSTEM "shared-constraints/allowed-values-control-group-property-name.ent">
+]>
+<METASCHEMA xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:meta="http://csrc.nist.gov/ns/oscal/metaschema/1.0"
+      xmlns="http://csrc.nist.gov/ns/oscal/metaschema/1.0" xsi:schemaLocation="http://csrc.nist.gov/ns/oscal/metaschema/1.0 ../../build/metaschema/toolchains/xslt-M4/validate/metaschema.xsd">
+      <schema-name>OSCAL Control Mapping Model</schema-name>
+      <schema-version>1.0.3</schema-version>
+      <short-name>oscal-mapping</short-name>
+      <namespace>http://csrc.nist.gov/ns/oscal/1.0</namespace>
+      <json-base-uri>http://csrc.nist.gov/ns/oscal</json-base-uri>
+      <remarks>
+            <p>The OSCAL Control mapping format can be used to describe how a collection of security controls and related control enhancements relate to another collection of controls. The root of the Control Catalog format is <code>mapping-collection</code>.
+            </p>
+      </remarks>
+      <import href="oscal_metadata_metaschema.xml"/>
+      <import href="oscal_mapping-common_metaschema.xml"/>
+
+      <define-assembly name="mapping-collection">
+            <formal-name>Mapping Collection</formal-name>
+            <description>A collection of control mappings.</description>
+            <root-name>mapping-collection</root-name>
+            <define-flag name="uuid" as-type="uuid" required="yes">
+                  <formal-name>Mapping Collection Universally Unique Identifier</formal-name>
+                  <description>A globally unique identifier with cross-instance scope for this catalog instance. This UUID should be changed when this document is revised.</description>
+            </define-flag>
+
+            <model>
+                  <assembly ref="metadata" min-occurs="1"/>
+                  <assembly ref="mapping" min-occurs="1" max-occurs="unbounded">
```

Comment:
If the root is a Catalog, we might encounter situations where some controls are not mapped to anything. Do we assume that a Resolved profile needs to be used to eliminate first the controls that have no mapping? there is min-occurs=1, and I interpret it as no control can exist without a mapping. Is it correct?

Reply:
No. The purpose of the mapping is to identify only the controls that are mapped. It doesn't say or infer anything about unmapped controls. I'll add some text in the docs about this.

The advantage of not specifying a baseline in the mapping is that any applicable mapping can be used with any baseline that incorporates these controls. I believe this increases the utility of community provided mappings, since custom mappings will not need to be produced for every baseline, which is a huge effort with little extra gain.

Reply:
Done in latest commit.

Also, the `min-occurs=1` requires that at least one `mapping` exists in a `mapping-collection` as it makes no sense to have a mapping without any mappings. 😊

Reply:
@david-waltermire-nist Indeed, a mapping-collection requires to have at least one mapping. The comment above was bringing forward the need identified also by @vikas-agarwal76 (above) of being able to note that some controls from the source catalog might not have corresponding (mappable) controls in the target catalog. If such controls are not listed in the mapping, tools will need to keep track of those controls by comparing the OSCAL mapping instance with the OSCAL catalog. I do not see it being easier.